example/pom.xml

```diff
@@ -119,6 +119,11 @@
       <artifactId>logback-classic</artifactId>
       <scope>runtime</scope>
     </dependency>
+    <dependency>
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
```

Comment:
I think we can remove this and use Netty `ByteBuf` or `ByteBufUtils` to do what this is being used for.